### PR TITLE
Reexport the other packages

### PR DESCRIPTION
**Pull request by AntonReinhard**
This PR adds Reexport.jl and exports all other currently released QED-project packages.
I wrote a small landing page for the docs that links the other projects' docs and repositories. QEDcore needs another release (v0.1.1) before its docs become available via the link.

### CODE REVIEW

**Review by SimeonEhrig — DISMISSED** [review]

File: docs/src/index.md [hunk 1]
```diff
@@ -4,7 +4,18 @@ CurrentModule = QED
 
 # QED
 
-Documentation for [QED](https://github.com/QEDjl-project/QED.jl).
+This is the documentation for [QED](https://github.com/QEDjl-project/QED.jl). It represents the combination of the following subpackages:
```
Comment (szabo137) [inline docs/src/index.md:7]:
```suggestion
This is the documentation for [`QED.jl`](https://github.com/QEDjl-project/QED.jl). It represents the combination of the following subpackages:
```

File: docs/src/index.md [hunk 2]
```diff
@@ -4,7 +4,18 @@ CurrentModule = QED
...
+
+**The two fundamental packages**:
+- [QEDbase](https://github.com/QEDjl-project/QEDbase.jl): Interfaces and some functionality on abstract types. [Docs](https://qedjl-project.github.io/QEDbase.jl/stable/)
```
Comment (szabo137) [inline docs/src/index.md:10]:
```suggestion
- [`QEDbase.jl`](https://github.com/QEDjl-project/QEDbase.jl): Interfaces and some functionality on abstract types. [Docs](https://qedjl-project.github.io/QEDbase.jl/stable/)
```

File: docs/src/index.md [hunk 3]
```diff
@@ -4,7 +4,18 @@ CurrentModule = QED
...
+- [QEDcore](https://github.com/QEDjl-project/QEDcore.jl): Implementation of core functionality that is needed across all or most content packages. [Docs](https://qedjl-project.github.io/QEDcore.jl/stable/)
```
Comment (szabo137) [inline docs/src/index.md:11]:
```suggestion
- [`QEDcore.jl`](https://github.com/QEDjl-project/QEDcore.jl): Implementation of core functionality that is needed across all or most content packages. [Docs](https://qedjl-project.github.io/QEDcore.jl/stable/)
```

File: docs/src/index.md [hunk 4]
```diff
@@ -4,7 +4,18 @@ CurrentModule = QED
...
+
+**The content packages**:
+- [QEDprocesses](https://github.com/QEDjl-project/QEDprocesses.jl): Scattering process definitions, models, and calculation of cross-sections and probabilities. [Docs](https://qedjl-project.github.io/QEDprocesses.jl/stable/)
```
Comment (szabo137) [inline docs/src/index.md:14]:
```suggestion
- [`QEDprocesses.jl`](https://github.com/QEDjl-project/QEDprocesses.jl): Scattering process definitions, models, and calculation of cross-sections and probabilities. [Docs](https://qedjl-project.github.io/QEDprocesses.jl/stable/)
```

File: docs/src/index.md [hunk 5]
```diff
@@ -4,7 +4,18 @@ CurrentModule = QED
...
+- [QEDevents](https://github.com/QEDjl-project/QEDevents.jl): Generation of random events for scattering processes. [Docs](https://qedjl-project.github.io/QEDevents.jl/stable/)
```
Comment (szabo137) [inline docs/src/index.md:15]:
```suggestion
- [`QEDevents.jl`](https://github.com/QEDjl-project/QEDevents.jl): Monte-Carlo event generation for scattering processes. [Docs](https://qedjl-project.github.io/QEDevents.jl/stable/)
```

File: docs/src/index.md [hunk 6]
```diff
@@ -4,7 +4,18 @@ CurrentModule = QED
...
+- [QEDfields](https://github.com/QEDjl-project/QEDfields.jl): Representation of background fields. [Docs](https://qedjl-project.github.io/QEDfields.jl/stable/)
```
Comment (szabo137) [inline docs/src/index.md:16]:
```suggestion
- [`QEDfields.jl`](https://github.com/QEDjl-project/QEDfields.jl): Description of classical electromagnetic fields used in background-field approximations. [Docs](https://qedjl-project.github.io/QEDfields.jl/stable/)
```

File: docs/src/index.md [hunk 7]
```diff
@@ -4,7 +4,18 @@ CurrentModule = QED
 
 # QED
```
Comment (szabo137) [inline docs/src/index.md:5]:
```suggestion
# QED.jl
```

**Review by szabo137 — APPROVED** [review]
Looks good for me - thanks for providing these changes 👌

I have just some minor suggestions. Please go ahead without additional review.

**Review by SimeonEhrig — APPROVED** [review]

**Pull request merged**